whazzup-core/src/main/java/io/github/whazzabi/whazzup/business/github/common/GithubRepositoryMatcher.java

```diff
@@ -0,0 +1,80 @@
+package io.github.whazzabi.whazzup.business.github.common;
+
+import io.github.whazzabi.whazzup.business.github.common.api.GithubRepo;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import static java.util.stream.Collectors.toList;
+
+public class GithubRepositoryMatcher {
+
+    private static final Logger LOG = LoggerFactory.getLogger(GithubRepositoryMatcher.class);
+
+    public static GithubRepositoryMatcher excludedArchivedRepos() {
+        return new GithubRepositoryMatcher().withIncludeArchivedRepos(false);
+    }
+
+    /**
+     * if specified the pr results will be filtered down to prs containing this keyword
+     */
+    private String repoNameRegex = "";
+    private Boolean includeArchivedRepos = true;
```

Comment:
should we set that one to false as default? I dont see a need for having the run status of archived repositories, do you see any?

Reply:
Nope, but I didn't want to change existing behaviour, even if nobody used that check before.